pages/library/_viewId.vue

```diff
@@ -103,6 +103,9 @@ export default Vue.extend({
     }
   },
   async beforeMount() {
+    this.$nextTick(() => {
```

Comment:
Same with this, we should adopt nuxt's original estimation system, which avoids us having to manually set loading on every single route (which is going to get annoying quickly, and makes maintainability worse)